ci/00_title.md

```diff
@@ -86,27 +70,61 @@ plugins: RevealMarkdown, RevealChalkboard, RevealHighlight, RevealMath.KaTeX, Re
 
 </style>
 
-# Continuous _What_ Now...?
+# Continuous Integration (CI)
 
-<div class="blockquote">
-  Automating the integration of code changes from multiple contributors into a single software project.
-<h4>&mdash; <a href="atlassian.com/continuous-delivery/continuous-integration">Atlassian</a></h4>
+<div class="blockquote-wrapper fragment">
+  <div class="blockquote">
+     It is the practice of integrating changes to a project made by individuals into a main, shared version frequently.
+    <h4>&mdash;https://book.the-turing-way.org</h4>
+  </div>
 </div>
 
+---
+
+<!-- .slide: data-state="blue_overlay yellow_flag yellow_strip purple_half_circle_bottom purple_blob right_e_top" data-background-video="./files/steampunk-85358.mp4" data-background-video-loop data-background-video-muted="true" data-auto-animate data-auto-animate-id="title" -->
+
+
+# Motivation
+How can we implement __automatic testing__ each time we push changes to the repository? 
```

Comment:
This sentence does not really fit the header "motivation".
Also, I find that in the font we are using, the boldface is so similar to standard text, that rather than creating emphasis, it becomes distracting, so I don't like using it.
Maybe I'll open a separate issue about this.

Reply:
```suggestion
# Motivation
We want to automatically test our code each time we push changes to the repository.
```

Reply:
How about it?